samples/apps/bookinfo/route-rule-all-v1.yaml

```diff
@@ -1,35 +1,10 @@
 type: route-rule
-name: productpage-default
-spec:
-  destination: productpage.default.svc.cluster.local
-  precedence: 1
-  route:
-  - tags:
-      version: v1
----
-type: route-rule
 name: reviews-default
 spec:
   destination: reviews.default.svc.cluster.local
   precedence: 1
   route:
   - tags:
       version: v1
+    weight: 100
```

Comment:
yes, I know, but it is easier to understand what the rule does if we include it explicitly.